src/renderer/containers/Settings/DolphinSettings.tsx

```diff
@@ -38,13 +38,18 @@ export const DolphinSettings: React.FC<{ dolphinType: DolphinLaunchType }> = ({
   const [resetModalOpen, setResetModalOpen] = React.useState(false);
   const [isResetType, setResetType] = React.useState<ResetType | null>(null);
   const { dolphinService } = useServices();
-  const { openConfigureDolphin, hardResetDolphin, softResetDolphin, importDolphin } = useDolphinActions(dolphinService);
+  const { openConfigureDolphin, hardResetDolphin, softResetDolphin, importDolphin, getDolphinUserPath } =
+    useDolphinActions(dolphinService);
 
   const dolphinIsReady = dolphinStatus === DolphinStatus.READY && !dolphinIsOpen && isResetType === null;
   const versionString: string =
     dolphinStatus === DolphinStatus.UNKNOWN ? "Not found" : !dolphinVersion ? "Unknown" : dolphinVersion;
 
   const openDolphinDirectoryHandler = async () => {
+    if (isMac || isLinux) {
+      const userSettingsPath = await getDolphinUserPath(dolphinType);
+      await window.electron.shell.openPath(userSettingsPath);
+    }
     await window.electron.shell.openPath(dolphinPath);
```

Comment:
Wait so on mac and linux it will open two folders? I don't really like that. Can we just open one folder, and call it `Open settings folder`?
```suggestion
      await window.electron.shell.openPath(userSettingsPath);
    } else {
      await window.electron.shell.openPath(dolphinPath);
    }
```

Reply:
i still worry about not being able to easily access the bundle folder :/ i really want to avoid making support difficult. I would honestly take anything that keeps it available. Maybe even a context menu on the button haha (this is a joke don't take this seriously)